qdrant-landing/themes/qdrant-2024/layouts/partials/core-features.html

```diff
@@ -1,27 +1,33 @@
 <section class="core-features">
-  <h2 class="core-features__title">{{ .Params.title }}</h2>
-  <p class="core-features__subtitle">{{ .Params.description }}</p>
-  <a class="link link_xl" href="{{ .Params.link.url }}">{{ .Params.link.text }}</a>
-  <div class="card core-features__cloud-native">
-    <div>
-      <h4 class="core-features__card-title">{{ .Params.cloudFeature.title }}</h4>
-      <p class="core-features__card-content">{{ .Params.cloudFeature.content }}</p>
-      <a class="link link_dark link_md" href="{{ .Params.cloudFeature.link.url }}">
-        {{ .Params.cloudFeature.link.text }}
-      </a>
+  <div class="container">
+    <h2 class="core-features__title mx-auto">{{ .Params.title }}</h2>
+    <p class="core-features__subtitle mx-auto">{{ .Params.description }}</p>
+    <div class="text-center">
+      <a class="link link_xl" href="{{ .Params.link.url }}">{{ .Params.link.text }}</a>
     </div>
-    <div class="core-features__cloud-native-image"></div>
-  </div>
-  <div class="core-features__cards">
-    {{ range .Params.featureCards }}
-    <div class="card core-features__card">
-      <div>
-        {{ .icon | safeHTML }}
-        <h6 class="core-features__card-title">{{ .title }}</h6>
-        <p class="core-features__card-content">{{ .content }}</p>
+    <div class="row-4 row-xl-12">
+      <div class="card core-features__cloud-native col-4 col-xl-12">
+        <div>
+          <h4 class="core-features__card-title">{{ .Params.cloudFeature.title }}</h4>
+          <p class="core-features__card-content">{{ .Params.cloudFeature.content }}</p>
+          <a class="link link_dark link_md" href="{{ .Params.cloudFeature.link.url }}">
+            {{ .Params.cloudFeature.link.text }}
+          </a>
+        </div>
+        <div class="core-features__cloud-native-image"></div>
+      </div>
+    </div>
+    <div class="core-features__cards row-4 row-xl-12">
```

Comment:
It's better to put cards inside `.col`.
`.row-xl-{n}` `.col-xl-{n}` are for bigger devices.
Something like this:
``` html
<div class="core-features__cards row-12 row-xl-4">
  <div class="col-12 col-xl-4">
    <div class="card core-features__card">
      <!-- more code -->
    </div>
  </div>
</div>
<!-- And so on -->
 ```    

Also, if we switch to the bootstrap grid layout we need to adjust some of the existing styles. For example, no need to fixate the width of `.core-features__card` etc.